Resources/Prototypes/Roles/Jobs/Civilian/clown.yml

```diff
@@ -23,10 +23,10 @@
           Piercing: 4
         groups:
           Burn: 3
-# DeltaV - Commenting out the clown snore sound because I am not fond of it (it makes me itchy and feral). By "I", I mean Leonardo_DaBepis.
-#    - type: SleepEmitSound
-#      snore: /Audio/Voice/Misc/silly_snore.ogg
-#      interval: 10
+    - type: SleepEmitSound
+      snore: /Audio/Voice/Misc/silly_snore.ogg
+      interval: 10
+    - type: Snoring # Necessary so SleepEmitSound sound effects play
```

Comment:
> That sound file triggers my misophonia and every time I hear it I want to scream and go feral, which is not conducive to MRP. This may seem like a strong emotional response to a noise but that's because it is.